src/__tests__/patients/new/NewPatient.test.tsx

```diff
@@ -45,128 +48,72 @@ describe('New Patient', () => {
             </TitleProvider>
           </Route>
         </Router>
+        <Toaster draggable hideProgressBar />
       </Provider>,
     )
-
-    wrapper.update()
-    return wrapper
   }
 
   beforeEach(() => {
     jest.resetAllMocks()
   })
 
   it('should render a general information form', async () => {
-    let wrapper: any
-    await act(async () => {
-      wrapper = await setup()
-    })
-
-    expect(wrapper.find(GeneralInformation)).toHaveLength(1)
+    setup()
+    expect(screen.getByText(/patient\.basicInformation/i))
   })
 
   it('should pass the error object to general information', async () => {
     const expectedError = { message: 'some message' }
-    let wrapper: any
-    await act(async () => {
-      wrapper = await setup(expectedError)
-    })
-    wrapper.update()
-
-    const generalInformationForm = wrapper.find(GeneralInformation)
-    expect(generalInformationForm.prop('error')).toEqual(expectedError)
+    setup(expectedError)
+    expect(screen.getByRole('alert')).toHaveTextContent(expectedError.message)
   })
 
   it('should dispatch createPatient when save button is clicked', async () => {
-    let wrapper: any
-    await act(async () => {
-      wrapper = await setup()
-    })
-
-    const generalInformationForm = wrapper.find(GeneralInformation)
-
-    act(() => {
-      generalInformationForm.prop('onChange')(patient)
-    })
-
-    wrapper.update()
-
-    const saveButton = wrapper.find('.btn-save').at(0)
-    const onClick = saveButton.prop('onClick') as any
-    expect(saveButton.text().trim()).toEqual('patients.createPatient')
-
-    await act(async () => {
-      await onClick()
-    })
-
-    expect(PatientRepository.save).toHaveBeenCalledWith(patient)
-    expect(store.getActions()).toContainEqual(patientSlice.createPatientStart())
-    expect(store.getActions()).toContainEqual(patientSlice.createPatientSuccess())
+    setup()
+    userEvent.type(screen.getByLabelText(/patient\.givenName/i), patient.givenName as string)
+    userEvent.click(screen.getByRole('button', { name: /patients\.createPatient/i }))
+    expect(PatientRepository.save).toHaveBeenCalledWith(
+      expect.objectContaining({ fullName: patient.fullName, givenName: patient.givenName }),
+    )
   })
 
-  it('should reveal modal (return true) when save button is clicked if an existing patient has the same information', async () => {
-    let wrapper: any
-    await act(async () => {
-      wrapper = await setup()
-    })
-
-    const saveButton = wrapper.find('.btn-save').at(0)
-    const onClick = saveButton.prop('onClick') as any
-    expect(saveButton.text().trim()).toEqual('patients.createPatient')
-
-    act(() => {
-      onClick()
-    })
-    wrapper.update()
-
-    expect(onClick()).toEqual(true)
-  })
+  // TODO: https://github.com/HospitalRun/hospitalrun-frontend/pull/2516#issuecomment-753378004
```

Comment:
Perhaps another one that needs a separate issue to be logged?

Reply:
I believe that is this one: https://github.com/HospitalRun/hospitalrun-frontend/issues/2546#issue-786389623